ccgains/trades.py

```diff
@@ -549,6 +556,28 @@ def append_ccgains_csv(
             skiprows=skiprows,
             default_timezone=default_timezone)
 
+    def append_binance_csv(
+            self, file_name, which_data='trades', delimiter=',',
+            skiprows=1,  default_timezone=tz.tzutc()):
```

Comment:
Everything good so far, but could you add a docstring here please? Just copy & adapt the one from append_poloniex_csv, then you'd merely have to add 'distribution' to the explanation of `which_data`.

Reply:
Done, and thanks for the constructive feedback!